app/src/main/res/layout/fragment_main.xml

```diff
@@ -42,7 +42,8 @@
             app:labelVisibilityMode="labeled"
             app:tabGravity="fill"
             app:itemPaddingTop="8dp"
-            app:tabIndicatorColor="@android:color/transparent" />
+            app:tabIndicatorColor="@android:color/transparent"
+            app:elevation="0dp"/>
```

Comment:
Almost there. We can also remove the two references to `@android:color/transparent` in this block, since they are not necessary, and also contribute to the issue. This will make the total of -1 lines added.

Reply:
Done!